### ISSUE TITLE
Github pages hates trailing slashes

### ISSUE THREAD

**Issue by leoyvens** [getzola/zola#171]
I tried hosting a Gutenberg website on github pages but it failed to load any css because Gutenberg appends a trailing slash, as in:
```html
<link rel="stylesheet" href="https://leodasvacas.github.io/poole.css/">
```
I had to manually remove the trailing slashes to get the site to work.

**Comment by MarcAstr0** [comment]
I had a similar problem with Nginx. I had to create a rewrite rule to remove the trailing slash from links to files:

```
rewrite ^/(.*\..*)/$ /$1 permanent;
```

**Comment by Keats** [comment]
Ha that's a bit annoying.
Is the problem only with assets urls or page urls as well? If it's only with the assets, are you generating the urls with the `get_url` global function?

**Comment by cldershem** [comment]
I don't think it's specific to github pages, I assume `poole.css` is a file and therefore, the slash makes the webserver think it should find an `index.html` to serve up. I cannot, however, find where the trailing slash is added.

**Comment by Keats** [comment]
It's coming from `Config::make_permalink` used by the `get_url` global fn: https://github.com/Keats/hyde/blob/master/templates/index.html#L15

That's the right thing to do for pages but not for assets. I guess a `trailing_slash` param to `get_url` could be added with a default of `true` but that you can turn off in the template. Would that solve the issue for everyone in the thread?

**Comment by leoyvens** [comment]
I'm using hyde, if it's fixed there it's fixed for me.

**Comment by piedoom** [comment]
Hi all, I was having a bit of trouble getting Rust to build Gutenberg from source (something about the windows compiler toolchain, nothing you have to worry about).  Anyways, I found a (very hacky) workaround.  Instead of

```
  <script src='{{ get_url(path="js/images.js") }}'></script>
```

you can write

```
  <script src='{{ get_url(path="js") }}images.js'></script>
```

Should hold out until the `next` branch releases.

**Comment by Keats** [comment]
Fixed in 0.3.0 that is currently building in travis/appveyor
